docs/DeveloperGuide.md

```diff
@@ -212,13 +196,30 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 This section describes some noteworthy details on how certain features are implemented.
 
 The features mentioned are:
-- [Adding contacts with optional arguments](#add-contacts-with-optional-arguments)
-- [Finding contacts by keywords](#find-contacts-by-keywords)
-- [Deleting multiple contacts by keywords](#delete-by-keywords)
-- [Importing a JSON file](#import-json-file)
-- [Undoing / redoing command](#proposed-undoredo-feature)
-- [Input Suggestion](#input-suggestion)
-- {and so on...}
+1. Viewing help
+2. Modify
+   1. Adding a person
+   2. Adding tags to persons
+   3. [Adding a remark](#adding-a-remark)
+   4. Editing a person
+   5. Deleting a person
+   6. [Deleting multiple person](#delete-multiple-persons)
+   7. Deleting tags from persons
+   8. Clearing all persons
+3. View
+   1. Listing all persons
+   2. [Finding persons](#finding-persons)
+   3. [Sorting persons](#sorting-persons)
+   4. Viewing statistics
+4. Share
+   1. [Importing contacts](#import-json-file)
+   2. [Exporting contacts](#export-json-file)
+5. Advance
+   1. [Aliasing commands](#aliasing-commands)
+6. Utility
+   1. [Input Suggestion](#input-suggestion)
+7. Exiting the program
+8. Saving the data
```

Comment:
I don't think we need to show the implementation for every feature hahah.  But we can edit this in future